package.json

```diff
@@ -11,20 +11,33 @@
     "range": "forta-agent run --range",
     "file": "forta-agent run --file",
     "publish": "forta-agent publish",
-    "test": "jest"
+    "test": "mocha ./src/bot-handlers/tests/liquidator-test.js && jest"
   },
   "dependencies": {
+    "@uma/common": "^2.7.0",
     "@uma/contracts-node": "^0.1.0",
+    "@uma/contracts-node-0-1-0": "npm:@uma/contracts-node@^0.1.0",
+    "@uma/core": "^2.10.0",
+    "@uma/financial-templates-lib": "^2.9.0",
     "bignumber.js": "^9.0.1",
     "ethers": "^5.4.6",
     "forta-agent": "^0.0.16",
```

Comment:
I suggest we update to `0.0.18` for the `initialize` handler and `FindingType.Info`